adafruit_tca9548a.py

```diff
@@ -81,6 +81,12 @@ def writeto(self, address, buffer, **kwargs):
             raise ValueError("Device address must be different than TCA9548A address.")
         return self.tca.i2c.writeto(address, buffer, **kwargs)
 
+    def writeto_then_readfrom(self, address, buffer_out, buffer_in, **kwargs):
+        """Pass thru for writeto_then_readfrom."""
+        """In linux, at least, this is a special kernel function call"""
```

Comment:
Travis isn't liking the two string literals back to back. Could make it one docstring, but I'd say just change this to:
```python
# In linux, at least, this is a special kernel function call
```